### PR TITLE
Add "I cannot find my repositories when trying to add a repository to Sider"

### PR DESCRIPTION
**Pull request by yykamei**

### CODE REVIEW

File: troubleshooting.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ This may be because Sider does not have access to your organization on Github. Y
 2. Find your organizations in the `Organization access`.
 3. Click `Grant`.
 
+## I cannot find my repositories when I add repository to Sider
+When your organization have migrated to GitHub Apps, you must include your repositories in "Repository access".
```
Comment (akito19) [inline troubleshooting.md]: Does this trouble happen when only migration to GitHub Apps?

Comment (ybiquitous) [inline troubleshooting.md]: [question] Sorry, I cannot understand "Repository access". 😓 What does it mean?

Comment (yykamei) [reply troubleshooting.md]: Yes, it happens to only organizations which have migrated to GitHub Apps.

Comment (yykamei) [reply troubleshooting.md]: It means the following section.

repository-access

Should I include this image on the section?

Comment (ybiquitous) [reply troubleshooting.md]: I understand! 👍

> Should I include this image on the section?

Yes, then I think users can understand it easily. 🙆‍♂️ 

And I think good also to include "GitHub" text like this: 👇 

```
..., you must include your repositories in "Repository access" on GitHub.
```

File: troubleshooting.md [hunk 2]
```diff
@@ -7,6 +7,12 @@ This may be because Sider does not have access to your organization on Github. Y
 2. Find your organizations in the `Organization access`.
 3. Click `Grant`.
 
+## I cannot find my repositories when I add repository to Sider
```
Comment (kvillaniholland) [inline troubleshooting.md]: I cannot find my repositories when trying to add a repository to Sider

File: troubleshooting.md [hunk 3]
```diff
@@ -7,6 +7,12 @@ This may be because Sider does not have access to your organization on Github. Y
...
+When your organization has migrated to GitHub Apps, you must include your repositories in "Repository access" on GitHub.
```
Comment (kvillaniholland) [inline troubleshooting.md]: If your organization has migrated to GitHub Apps, you must include your repositories in the "Repository access" section on GitHub.

File: troubleshooting.md [hunk 4]
```diff
@@ -7,6 +7,12 @@ This may be because Sider does not have access to your organization on Github. Y
...
+Visit [this page](https://github.com/apps/sider/installations/new) and select your organization, then you can configure repository access of your organization.
```
Comment (kvillaniholland) [inline troubleshooting.md]: and select your organization. From there, you can configure repository access for your organization.

**Pull request merged**